docs/content/preview/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -215,13 +215,17 @@ For details on the architecture design, see [Automatic resharding of data with t
 
 #### Enable automatic tablet splitting
 
-To enable automatic tablet splitting, use the `yb-master` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-master/#enable-automatic-tablet-splitting) flag and specify the associated flags to configure when tablets should split, and use `yb-tserver` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-tserver/#enable-automatic-tablet-splitting). The flag usage must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
+When automatic tablet splitting is enabled, newly-created tables with [hash sharding](../../../architecture/docdb-sharding/sharding/#hash-sharding) have one tablet per node by default.
 
-When automatic tablet splitting is enabled, newly-created tables with [hash sharding](../../../architecture/docdb-sharding/sharding/#hash-sharding) have one shard **per _yb-tserver_** by default; from version `2.14.10` such tables have one shard (for servers with up to 2 CPU cores) or two shards (for servers with up to 4 CPU cores) **per _cluster_** by default.
+In addition, from version 2.14.10, for servers with up to 2 CPU cores, newly-created tables have one tablet per table, and servers with up to 4 CPU cores have 2 tablets per table.
```

Comment:
"newly-created tables have one tablet per table"
It's strange we're expressing the number of tablets in terms of the number of tables while talking about a single table creation. How about:

For servers with up to 2 CPU cores, newly-created tables will have a single tablet. For servers with up to 4 CPU cores, newly-created tables will have 2 tablets.

Reply:
I think it is to make clear the distinction that it is NOT per node (previous paragraph). Because the previous paragraph specified 1 tablet per node, it may be interpreted ambiguously without per table here. @arybochkin

Reply:
That's why _per cluster_ was used previously..

Reply:
We can standardize on per cluster if you like - there was a mix before

Reply:
I'm ok to go with proposed changes.